model_compression_toolkit/qat/common/qat_get_quantizer_config.py

```diff
@@ -0,0 +1,106 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import List
+from model_compression_toolkit.core import common
+from model_compression_toolkit.quantizers_infrastructure.common.base_trainable_quantizer_config import \
+    TrainableQuantizerWeightsConfig, TrainableQuantizerActivationConfig, TrainableQuantizerCandidateConfig
+
+
+def get_trainable_quantizer_weights_config(
+        n: common.BaseNode,
+        weights_quantization_candidates: List[TrainableQuantizerCandidateConfig] = None) -> TrainableQuantizerWeightsConfig:
+    """
+    Returns the relevant configurations for weights trainable quantizer
+
+    Args:
+        n: BaseNode
+
+    Returns:
+         TrainableQuantizerWeightsConfig: an object that contains the quantizer configuration
+    """
+    final_cfg = n.final_weights_quantization_cfg
+    return TrainableQuantizerWeightsConfig(final_cfg.weights_quantization_method,
+                                           final_cfg.weights_n_bits,
+                                           final_cfg.weights_quantization_params,
+                                           final_cfg.enable_weights_quantization,
+                                           final_cfg.weights_channels_axis,
+                                           final_cfg.weights_per_channel_threshold,
+                                           final_cfg.min_threshold,
+                                           weights_quantization_candidates)
+
+
+def get_trainable_quantizer_activation_config(
+        n: common.BaseNode,
+        activation_quantization_candidates: List[TrainableQuantizerCandidateConfig] = None) -> TrainableQuantizerActivationConfig:
+    """
+    Returns configurations for activation trainable quantizer
+
+    Args:
+        n: BaseNode
+
+    Returns:
+         TrainableQuantizerActivationConfig - an object that contains the quantizer configuration
+    """
+    final_cfg = n.final_activation_quantization_cfg
+    return TrainableQuantizerActivationConfig(final_cfg.activation_quantization_method,
+                                              final_cfg.activation_n_bits,
+                                              final_cfg.activation_quantization_params,
+                                              final_cfg.enable_activation_quantization,
+                                              final_cfg.min_threshold,
+                                              activation_quantization_candidates)
+
+
+def get_trainable_quantizer_quantization_candidates(n: common.BaseNode):
+    """
+    Returns quantization configuration candidates for activation and weights trainable quantizer
```

Comment:
Add to the description an explanation (or at least some mentioning) about the validations that this function preforms, this is an important part of its purpose.